usr/share/rear/lib/layout-functions.sh

```diff
@@ -530,8 +530,7 @@ get_disk_size() {
 
     local block_size=$(get_block_size ${disk_name%/*})
 
-    [ -r /sys/block/$disk_name/size ]
-    BugIfError "Could not determine size of disk $disk_name, please file a bug."
+    $(retry_command test -r /sys/block/$disk_name/size) || Error "Could not determine size of disk $disk_name"
```

Comment:
Removing `$(` and `)` would damage receiving parameter. So instead:
```
devsize='8589934592'
```
you will get something like
```
devsize='
8589934592'
```